packages/react/src/recipes/FlexTemplate/FlexSection/FlexSection.tsx

```diff
@@ -0,0 +1,1011 @@
+            <Grid className={styles.normalizeMargin}>
+              <Grid.Column>
+                <BreakoutBanner
+                  className={clsx(styles.wrapper, styles.normalizeMargin)}
+                  align={breakoutBanner.fields.align}
+                  backgroundColor={backgroundColor}
+                  backgroundImageSrc={{
+                    narrow: lightNarrowBg,
+                    regular: lightWideBg,
+                    wide: lightWideBg,
+                  }}
+                  backgroundImagePosition={{
+                    narrow: 'bottom',
+                    regular: 'right',
+                    wide: 'right',
+                  }}
+                  {...(breakoutBanner.fields.showLogo && {
+                    leadingVisual: (
+                      <Box className={styles.logoWrapper}>
+                        <ForresterResearch />
+                      </Box>
+                    ),
+                  })}
+                >
+                  <BreakoutBanner.Heading as={breakoutBanner.fields.headingLevel}>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                    {breakoutBanner.fields.showFootnotes ? (
+                      <>
+                        {' '}
+                        <InlineLink
+                          id="inline-link-2"
+                          href="#footnote-2"
+                          className={clsx(styles.footnoteInlineLink, styles.footnoteSizeLarge)}
+                          aria-label="Footnote 2"
+                        >
+                          2
+                        </InlineLink>
+                      </>
+                    ) : null}
+                  </BreakoutBanner.Heading>
+                  <BreakoutBanner.LinkGroup>
+                    <Link href="#">Learn more</Link>
+                  </BreakoutBanner.LinkGroup>
+                </BreakoutBanner>
+              </Grid.Column>
+            </Grid>
+          )}
+
+          {statistics && (
+            <Grid className={styles.normalizeMargin} fullWidth>
+              {Array(statistics.fields.count)
+                .fill(null)
+                .map((_, i) => (
+                  <Grid.Column span={{medium: 6, large: statistics.fields.count === 3 ? 4 : 3}} key={i}>
+                    <Statistic
+                      className={styles.statistic}
+                      variant={statistics.fields.variant}
+                      size={statistics.fields.size}
+                      style={{
+                        backgroundColor:
+                          statistics.fields.variant === 'boxed'
+                            ? `var(--brand-color-canvas-${backgroundColor === 'default' ? 'subtle' : 'default'})`
+                            : undefined,
+                      }}
+                    >
+                      <Statistic.Heading as="p">
+                        $2M+
+                        {statistics.fields.showHeadingFootnotes ? (
```

Comment:
@joshfarrant FYI these aren't rendering correctly. Let me know if I'm misunderstanding something though.


<table>
<tr>
<th> Dotcom flexsection </th> <th> This PR </th>
</tr>
<tr>
<td valign="top">

<img width="1533" height="357" alt="Screenshot 2025-10-02 at 11 19 51" src="https://github.com/user-attachments/assets/c6cb0b0b-445e-4461-8176-a2ad09ed3c78" />



 </td>
<td valign="top">


<img width="1362" height="347" alt="Screenshot 2025-10-02 at 11 18 56" src="https://github.com/user-attachments/assets/098397cb-85a7-4af5-a3d0-560e2796fd7b" />


</td>
</tr>
</table>

Reply:
Update to ☝️, I just spotted the issue now, so you can ignore some of the above. It's the footnotes thing 😞. It looks like passing a Fragment doesn't work either, which is weird. That bug will need to be fixed before we can enable an inline foot note like this. Fine to disable for now 👍